### PR TITLE
Feature/UI-10 migrate List

### PR DESCRIPTION
**Pull request by PiotrGrzechnik**

### CODE REVIEW

File: packages/list/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Talixo Switcher
```
Comment (rangoo94) [inline packages/list/README.md]: Update this `README` file or use `npm run create`

File: packages/list/src/List.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ * Component which represents List.
+ *
+ * @param {object} props
+ * @param {string} [props.className]
+ * @returns {React.Element}
+ */
+function List (props) {
+  const { bullet, children, className, style, ...passedProps } = props
+
+  const mappedChildren = React.Children.map(children, child => {
```
Comment (rangoo94) [inline packages/list/src/List.js]:
```js
const mappedChildren = React.Children.map(children, child => (
  <li>
    <span>{bullet}</span>
    {child}
  </li>
))
```

File: packages/list/src/List.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return (
+      <li>
+        <span>{bullet}</span>
+        {child}
+      </li>
+    )
+  })
+  return (
+    <ul className={cls(name, className)} {...passedProps} style={style}>
```
Comment (rangoo94) [inline packages/list/src/List.js]: `style` will be passed anyway by `{...passedProps}`

Comment (michalochman) [reply packages/list/src/List.js]: Provided that it's not destructured anymore ;)

File: packages/list/styles/geometry.sass [hunk 4]
```diff
@@ -0,0 +1,11 @@
+@import './config'
+
+@include module('list')
+  list-style-type: none
+
+  > li > *
```
Comment (rangoo94) [inline packages/list/styles/geometry.sass]: Why `li > *`? Could you simplify it?

**Pull request merged**